packages/sources/coinmarketcap/src/transport/crypto.ts

```diff
@@ -79,14 +82,13 @@ export const httpTransport = new HttpTransport<HttpTransportTypes>({
       }
     }
 
-    for (const [idType, fullList] of Object.entries(groupedParams)) {
-      if (fullList && fullList.length > 0) {
-        // CMC does not support more than 120 unique quotes
-        const chunkedList = splitArrayIntoChunks(fullList, 120)
-
-        // This could be further optimized in cases with more than 120 entries, to make sure that
-        // chunkes are grouped optimally to avoid sending unnecessary converts
```

Comment:
We may want to keep this comment since it still applies. We're still potentially sending extra bases for each quote. Just so we know for future improvements if we ever want to make them

Reply:
Added similar comment but for the base since 120 quote limit is not relevant anymore

Reply:
That's perfect